### PR TITLE
Create BLorient12510

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient12510 based on the description in Strelcyn's catalogue but without access to images. I encoded the parts of the Gospel that are marked in the manuscript to be read on each day of the week as ms_item, as it is described in the catalogue.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient12510.xml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                                <desc>A drawing in black ink representing 
+                                    <ref type="authFile" corresp="AT1035HolyMan">St Takla Hāymānot with three pairs of wings</ref>.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#3r"/>
+                                    <desc type="OwnershipNote">The name of the first owner is erased, the second was inscribed as
+                                        <persName ref="PRS14328ZawaldaMaryam">Zawalda Māryām</persName>.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient12510.xml:139]: Sure that it is not Walda Māryām?

File: LondonBritishLibrary/orient/BLorient12510.xml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                                <decoNote xml:id="b5" type="Other"><term key="leafStringMark">small pieces of violet, green, and yellow yarn</term>
+                                    </decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1900" notAfter="1961">First half 20th century</origDate>
+                        </origin>
+                        <provenance>According to an inscription on the front flyleaf, it was presented by the Wellcome Historical Medical Library,
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient12510.xml:169]: No further information on the provenance is available, i.e. in the introduction of the catalogue or so?

**Review by thea-m — APPROVED** [review]
Thank you, very nice

**Comment by eu-genia** [comment]
We never create msItem in lectionaries for the days of the week
we usually describe the Gospels as Gospels and add the information on the days of the week as findingAid

If we change our approach then a lot of records have to be revised

I would go with the established practice

File: LondonBritishLibrary/orient/BLorient12510.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient12510.xml:169]: As far as I can see, No. 5 was not mentioned in the Preface.

**Comment by CarstenHoffmannMarburg** [comment]
> We never create msItem in lectionaries for the days of the week we usually describe the Gospels as Gospels and add the information on the days of the week as findingAid
> 
> If we change our approach then a lot of records have to be revised
> 
> I would go with the established practice

Me too, I agree, that this is not very useful. I just followed Strelcyn's description. I will rework it and describe them as findingAids.

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**